core/src/main/java/io/fabric8/maven/core/util/SpringBootProperties.java

```diff
@@ -23,5 +23,9 @@
     public static final String SERVER_PORT = "server.port";
     public static final String SERVER_KEYSTORE = "server.ssl.key-store";
     public static final String DEV_TOOLS_REMOTE_SECRET = "spring.devtools.remote.secret";
+    public static final String DEV_TOOLS_REMOTE_SECRET_ENV = "SPRING_DEVTOOLS_REMOTE_SECRET";
     public static final String CONTEXT_PATH = "server.context-path";
+    public static final String SPRING_BOOT_GROUP_ID = "org.springframework.boot";
+    public static final String SPRING_BOOT_ARTIFACT_ID = "spring-boot";
+    public static final String SPRING_BOOT_DEVTOOLS_ARTIFACT_ID = "spring-boot-devtools";
```

Comment:
[3] Ok so let is here for now. I thought even to move this out into a complete new project so that it could be referenced e.g. from d-m-p, too, for doing the generation stuff. But this is an even bigger effort.